### PR TITLE
Snippet coding style

### PR DESCRIPTION
**Pull request by rainemak**
Fix coding style as per https://wiki.qt.io/Coding_Conventions

### CODE REVIEW

File: Develop/Apps/Tutorials/Combining_C++_with_QML/README.md [hunk 1]
```diff
@@ -19,6 +19,12 @@ Combining C++ and QML requires three distinct steps. First we need to create a d
 
 Exposing data from C++ into QML is done via the [Qt Model/View framework](http://doc.qt.io/qt-5/model-view-programming.html). Our model is a simple array of strings. Qt provides a [QStringListModel](http://doc.qt.io/qt-5/qstringlistmodel.html) for this use case but for educational purposes we will provide our own by inheriting from a [QAbstractListModel](http://doc.qt.io/qt-5/qabstractlistmodel.html). The relevant parts of the model header file look like this.
 ```cpp
+#ifndef DEMOMODEL_H
+#define DEMOMODEL_H
+
+#include <QObject>
```
Comment (vigejolla) [inline Develop/Apps/Tutorials/Combining_C++_with_QML/README.md:25]: The [source repository](https://github.com/sailfishos/sample-app-cppqml/blob/master/src/demomodel.h) does not have this include, please add it there as well

File: Develop/Apps/Tutorials/Combining_C++_with_QML/README.md [hunk 2]
```diff
@@ -35,10 +35,10 @@ public:
 
     explicit DemoModel(QObject *parent = 0);
 
-    virtual int rowCount(const QModelIndex&) const { return backing.size(); }
-    virtual QVariant data(const QModelIndex &index, int role) const;
+    int rowCount(const QModelIndex&) const override { return backing.size(); }
+    QVariant data(const QModelIndex &index, int role) const override;
```
Comment (vigejolla) [inline Develop/Apps/Tutorials/Combining_C++_with_QML/README.md:39]: Similarly here; all these changes are good but please add them to the source repo as well

**Review by vigejolla — APPROVED** [review]
LGTM, but please make the changes to the source repo as well

File: Develop/Apps/Tutorials/Combining_C++_with_QML/README.md (hunk 1 above)
Comment (rainemak) [reply Develop/Apps/Tutorials/Combining_C++_with_QML/README.md:25]: I cleaned it up from my sources as well but forgot to copy to the docs repo. So, not needed and will clean it up for here as well.

File: Develop/Apps/Tutorials/Combining_C++_with_QML/README.md (hunk 2 above)
Comment (rainemak) [reply Develop/Apps/Tutorials/Combining_C++_with_QML/README.md:39]: Thanks for reminding, will create a PR.

Comment (rainemak) [reply Develop/Apps/Tutorials/Combining_C++_with_QML/README.md:39]: Fixes to the source repo are here https://github.com/sailfishos/sample-app-cppqml/pull/4

**Review by vigejolla — APPROVED** [review]
nitpick: typo in commit message (contructor vs constructor). Otherwise LGTM

**Pull request merged**